### ISSUE TITLE
Add detonation time for bombs

### ISSUE THREAD

**Issue by enjoybeta** [BahamutoD/BDArmory#50]
Bombs often collide while i dropping them by my bomber, i hope you can implement the code to enable us to chose time for a bomb to wait forable to be detonated. like 2s, then it's guaranteed to leave cargo area

**Comment by boomchacle** [comment]
I agree with you. it could be like the one on the missiles that prevents them from firing off untill a certain amount of time.  (Drop time)

**Comment by TangentFoxy** [comment]
In fact, it should literally be the drop time, just copied over.

I was gonna suggest opening this issue on the fork of the guy who was taking on support for BDA while Bahamuto is away, but I don't know the name and can't find him. :/

**Comment by boomchacle** [comment]
ask harpwner53, he already does this
